### ISSUE TITLE
Bank card showing first three transaction

### ISSUE THREAD

**Issue by MominRaza** [MominRaza/messages_wallet#11]
### Summary

Bank card in accounts screen showing first three transaction instead of last three transaction

### Steps to Reproduce

1. See bank card which has more than three transactions

### Expected Behavior

Should show last three transactions

### Actual Behavior

Showing first three transactions

**Comment by NeelVora1119** [comment]
Hello MominRaza,
can I help to solve this issue?

**Comment by MominRaza** [comment]
@NeelVora1119 Yes please, if you need any help feel free to ping me on Twitter/X
